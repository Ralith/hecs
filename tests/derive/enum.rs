use hecs::{Bundle, Query};

#[derive(Query)]
enum Foo {}

#[derive(Bundle)]
enum Bar {}

fn main() {}
