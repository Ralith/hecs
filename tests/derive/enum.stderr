error: derive(Query) may only be applied to structs
 --> $DIR/enum.rs:4:6
  |
4 | enum Foo {}
  |      ^^^

error: derive(Bundle) does not support enums or unions
 --> $DIR/enum.rs:7:6
  |
7 | enum Bar {}
  |      ^^^
